openedx/core/lib/xblock_utils.py

```diff
@@ -302,7 +304,9 @@ def add_staff_markup(user, has_instructor_access, disable_staff_debug_info, bloc
         # check that the course is a mongo backed Studio course before doing work
         is_studio_course = block.course_edit_method == "Studio"
 
-        if is_studio_course:
+        not_ccx_course = CustomCourseForEdX.objects.filter(course_id=course_id)
+        # will not return the View Unit in Studio button when viewing a CCX course
+        if is_studio_course and not_ccx_course:
```

Comment:
I think the language around this section could use some work. `not_ccx_course` describes what we're looking for, but it doesn't give us context as to why.